### ISSUE TITLE
Bishop Aqewŝma 

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#271]
Is there a URI for Bishop Aqewŝma, how should I transliterate his name?

<tei:div xmlns:tei="http://www.tei-c.org/ns/1.0">
        <idno xmlns="http://www.tei-c.org/ns/1.0" type="URI">http://syriaca.org/person/1291/tei</idno>
        <persName xmlns="http://www.tei-c.org/ns/1.0" xml:id="name1291-2" xml:lang="en">Ithalaha, the deacon</persName>
        <note xmlns="http://www.tei-c.org/ns/1.0" type="abstract">In hagiography: Ithalaha was a
          deacon who was martyred with the Bishop Aqewŝma during the persecution of Shapur.</note>
      </tei:div>

**Comment by jsaintlauren** [comment]
We need a record for Joseph ..

**Comment by jsaintlauren** [comment]
We have two records for Bishop A., 1558 and 1135.  

@nathangibson please merge 1558 and 1135

**Comment by nathangibson** [comment]
Merge is done, leaving the rest to you @jsaintlauren

**Comment by jsaintlauren** [comment]
Can you explain please the display of the description? @nathangibson of 
http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/1135 ?

**Comment by nathangibson** [comment]
@jsaintlauren Do you mean the fact that the "abstract" and "description" are similar? I merged person/1135 and person/1558 using the script, so now it is on Dave's list to check and clean up abstracts. That's why you're seeing double.

**Comment by jsaintlauren** [comment]
Yes .. thanks.  I was confused by the "in hagiography" prefix being attached to both abstract and description.  I don't know if 1135 and 886 are the same...

**Comment by wlpotter** [comment]
It looks like the abstract and description still need to be cleaned up:

```
<note xml:lang="en" type="abstract" xml:id="abstract-en-1135">In hagiography: ʿAqewshma was a bishop from
          the region of <placeName ref="http://syriaca.org/place/993">Adiabene</placeName> who was
          martyred during the persecution of Shapur II and buried by an Armenian princess.</note>
<note xml:lang="en" subtype="auto-formatted" source="#bib1135-2" type="description">In hagiography: <persName ref="http://syriaca.org/person/1135">ʿAqebshma</persName> is a martyr
            from <placeName ref="http://syriaca.org/place/526">Persia</placeName> who was killed with
                        Joseph and <persName ref="http://syriaca.org/person/1291">Ithalaha</persName>. </note>
```

https://syriaca.org/person/1135 and https://syriaca.org/person/886 are unmerged at present (if they turn out to be the same, cf. #433)
